easybuild/easyconfigs/p/PDBFixer/PDBFixer-1.7-foss-2020b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'PDBFixer'
+version = '1.7'
+
+homepage = 'https://github.com/openmm/pdbfixer'
+description = """PDBFixer is an easy to use application for fixing problems in Protein Data Bank files in preparation
+for simulating them."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/openmm/pdbfixer/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('OpenMM', '7.5.1'),
```

Comment:
As AlphaFold uses a patched OpenMM and PDBFixer depends on OpenMM, should we skip the separate PDBFixer and add it into AlphaFold?

Reply:
Yeah, that makes sense, I'll change that.